recipes/opencv/4.x/conanfile.py

```diff
@@ -713,8 +728,10 @@ def package_info(self):
         def get_lib_name(module):
             if module == "ippiw":
                 return "%s%s" % (module, debug)
-            prefix = "" if module in ("correspondence", "multiview", "numeric") else "opencv_"
-            return "%s%s%s%s" % (prefix, module, version, debug)
+            elif module in ("correspondence", "multiview", "numeric"):
+                return module
```

Comment:
are you sure they don't have debug postfix?

Reply:
The picture is the result of my compilation with this parameter.
`conan create -k . opencv/4.5.5@ -o opencv:contrib=True -o opencv:contrib_sfm=True -o opencv:with_ipp=opencv-icv -s build_type=Debug --build=missing`
![image](https://user-images.githubusercontent.com/3735828/160120828-0cb0a729-8b42-47d6-a44e-5aaecce08a4d.png)